classes/Link.php

```diff
@@ -1527,6 +1527,20 @@ public static function getUrlSmarty($params)
                     $params['type'] = (isset($params['type']) ? $params['type'] : null)
                 );
 
+                break;
+            case 'manufacturerImage':
+                $link = $context->link->getManufacturerImageLink(
+                    (int) $params['id'],
+                    $params['type'] = (isset($params['type']) ? $params['type'] : null)
```

Comment:
It still weird to see this param attribution everywhere :sweat_smile:

Reply:
I totally agree ... I'm wondering if i could do something here or open another PR?

Reply:
Maybe in another PR, it will be easier as this one has already been approved.